ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -1157,6 +1229,63 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-gcp
+- as: habana-notebooks-e2e-tests
+  run_if_changed: (base\/ubi8-python-3.8\/*)|(jupyter\/minimal\/ubi8-python-3.8\/*)|(jupyter\/datascience\/ubi8-python-3.8\/*)|(habana\/*)
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: habana-jupyter-1.9.0-ubi8-python-3.8-test-e2e
+      cli: latest
+      commands: |
+        oc new-project habana-jupyter-1.9.0-ubi8-python-3-8
+        IFS=':' read -r -a NOTEBOOK_IMAGE <<< "${JUPYTER_HABANA_IMAGE}"
+        make deploy8-habana-1.9.0-ubi8-python-3.8 \
+            -e IMAGE_REGISTRY="${NOTEBOOK_IMAGE[0]}" -e NOTEBOOK_TAG="${NOTEBOOK_IMAGE[1]}"
+        make test-habana-1.9.0-ubi8-python-3.8
+        make undeploy8-habana-1.9.0-ubi8-python-3.8
+      dependencies:
+      - env: JUPYTER_HABANA_IMAGE
```

Comment:
I think we need to specify the version to each of habana image on ENV.Otherwise it will have conflict

Reply:
ack, updated.